Wox.Infrastructure/Alphabet.cs

```diff
@@ -15,17 +15,26 @@ public static class Alphabet
         private static readonly HanyuPinyinOutputFormat Format = new HanyuPinyinOutputFormat();
         private static ConcurrentDictionary<string, string[][]> PinyinCache;
         private static BinaryStorage<ConcurrentDictionary<string, string[][]>> _pinyinStorage;
-        private static Settings _settings;
```

Comment:
If we remove this object reference, this would mean the state of 'ShouldUsePinyin' is only set at start up. What happens when a user changes the ShouldUsePinyin option during runtime?

Reply:
You're totally right! I'll fix that.